api-docs/influxdb/v2/v2/ref.yml

```diff
@@ -12752,6 +12752,12 @@ paths:
 
              - Returns this error only if the [Go (golang) `ioutil.ReadAll()`](https://pkg.go.dev/io/ioutil#ReadAll) function raises an error.
              - Returns `Content-Type: application/json` for this error.
+        '422':
+          content:
+            application/json:
+              schema:
+               $ref: '#/components/schemas/Error' 
+          description: The request was well-formed, but some or all the points were rejected due to semantic errors--for example, schema conflicts or retention policy violations. Error message contains details for one or more rejected points.
```

Comment:
Spaces between words and dashes in : `errors--for`

Reply:
> Spaces between words and dashes in : `errors--for`

It represents and em-dash and [doesn't get spaces](https://developers.google.com/style/dashes#em-dashes).

Reply:
I do appreciate the keen eye, though.